entity-table/index.js

```diff
@@ -182,38 +182,57 @@ module.exports = {
 			);
 		});
 	},
-	tableOldNewProcessor: function(options) {
+	tableOldNewProcessor: function(optionsIn) {
+		let options = Object.assign({ kinesisBatchLimit: 200}, optionsIn);
 		let self = this;
 		return function(event, context, callback) {
 			let streams = {};
+			let batchSize = event.Records.length;
+			let filePrefix = event.Records[0].eventID;
+			logger.info(`batchSize=${batchSize}`);
+			let localReadOpts = Object.assign({
+				prefix: filePrefix,
+				toCheckpoint: { force: true}
```

Comment:
@jgrantr This isn't correct.  I believe that `force` should be a top level field.